sources/visual/SoundRect.cpp

```diff
@@ -332,12 +332,13 @@ void SoundRect::setUps()
 //    lastY = (float)y;
 //}
 
-void SoundRect::associateSound(double *theBuff, unsigned long buffFrames, unsigned int buffChans)
+void SoundRect::associateSound(double *theBuff, unsigned long buffFrames, unsigned int buffChans, string name)
```

Comment:
Passage de chaîne sans copie

```suggestion
void SoundRect::associateSound(double *theBuff, unsigned long buffFrames, unsigned int buffChans, const string &name)
```